public/app/features/alerting/unified/mocks/alertmanagerApi.ts

```diff
@@ -38,3 +45,17 @@ export function mockAlertmanagerConfigResponse(
     )
   );
 }
+
+export const alertmanagerAlertsListHandler = () =>
+  http.get('/api/alertmanager/:datasourceUid/api/v2/alerts', () =>
+    HttpResponse.json([
+      mockAlertmanagerAlert({
```

Comment:
I think it would be nice if the defaults reflected real server responses.
I'd provide a few alerts of each type. If an alert is suppresed `silencedBy` should return an ID of a silence returned in by the silences handler

Reply:
I've linked them together a bit more now - using a common const to link the silence ID together. Is that okay with you for now?